mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import {PropsWithChildren, ReactNode} from "react";
+import clsx from "clsx";
+import {PositionTooltip} from "src/component/tooltip/PositionTooltip";
+import styles from "src/component/tooltip/Tooltip.module.scss";
+
+/**
+ * Describe props component Tooltip
+ */
+interface TooltipProps {
+  /**
+  * Tooltip's content
+  */
+  content: string | ReactNode;
+  /**
+  * Tooltip's className
+  */
+  className?: string;
+  /**
+  * Tooltip's position
+  * default: {@link PositionTooltip.TOP}
+  */
+  position?: PositionTooltip;
+}
+
+export const Tooltip: React.FC<PropsWithChildren<TooltipProps>> = (props: PropsWithChildren<TooltipProps>) => {
+  const classes = clsx(styles.tooltip, props.className && styles[props.className], styles[props.position ?? PositionTooltip.TOP]);
```

Comment:
What does second parameter do? Did you check it?

Reply:
The second condition implies the presence of a classname prop, and if it is a formalized style conveyed in this prop, if not, then it simply skips it

Reply:
If you will not get prop className then styles[props.className]  will not be available

Reply:
Right! Notice, ``classname`` is optional type, custom style. Default style is `tooltip`